book/src/requirements.md

```diff
@@ -13,6 +13,11 @@ It is required to use Clang 5.0 or greater.
 
 #### Windows
 
+Recent versions of Windows 10 or Windows 11:
+```
```

Comment:
You have to add a language hint here so `mdbook` knows this is not rust code. I think `powershell` would be adequate but I don't use windows myself so I'm not sure.

Reply:
Yeah, I think powershell should be fine, at least I remember using it for syntax highlighting in VS Code's md support.